### DISCUSSION TITLE
Is there any way to run this repo on TPU?

### DISCUSSION THREAD

**Discussion by bamps53** [EmGarr/kerod#7] [Q&A]
**Is your feature request related to a problem? Please describe.**
I saw the notebook which has TPU in its name, but now I can't find.
Is there any way to train DETR on TPU?
Thanks,

**Comment by EmGarr** [comment]
I tried in the past on faster-rcnn but the TPU  support  on tensorflow 2.0 was experimental and I haven't been able to run a training. 
I think it should be fairly easy now.  If you can just try using the TPUStrategy and let me know.

**Comment by bamps53** [comment]
Good, I'll try it after testing on GPU.

**Comment by nisarahamedk** [comment]
> Good, I'll try it after testing on GPU.

Were you able to train DETR on TPU?
